src/autopas/cells/FullParticleCell.h

```diff
@@ -86,6 +87,16 @@ class FullParticleCell : public ParticleCell<Particle> {
    */
   SoA<SoAArraysType> _particleSoABuffer;
 
+  /**
```

Comment:
should be present only if cuda is enabled

Reply:
@julian-spahl ifdef this!

Reply:
typename Particle::CudaDeviceArraysType is empty when cuda is not enabled.